src/main.c

```diff
@@ -1,17 +1,27 @@
-
 #include <stdbool.h>
 #include <stdio.h>
+#define __USE_GNU
```

Comment:
Could we use this GNU version instead of the one being used in util.c?

https://github.com/8dcc/tf2-cheat/blob/8faf4422981d36551a5d7924e5139bb8bdbb88ba/src/util.c#L81-L96

Reply:
What do you mean?

Reply:
Never mind. I declare a `our_link_map` structure inside `find_sig()` because I need to use `link->phdr[0].p_memsz` for getting the size of the module. I misread and I thought you were using a different version of `link_map` instead of `Dl_info`.